server/src/main/java/org/apache/iotdb/db/metadata/mnode/StorageGroupEntityMNode.java

```diff
@@ -81,6 +81,11 @@ public boolean isStorageGroup() {
     return true;
   }
 
+  @Override
+  public MNodeType getMNodeType(Boolean isConfig) {
+    return MNodeType.STORAGE_GROUP;
+  }
```

Comment:
It seems the ```StorageGroupEntityMNode``` is only described as storage group? What if the user want to check whether it is device? Or it's just a rule and we don't suggest user to reach into such case?

Reply:
Yes, in current mpp node management framework, it is a little bit hard to define one node both as sg and device. If user want to know whether it is device, maybe need to try another sql like show child paths for this path to see whether has measurement child node. I will imporve it from the next pr.